yt_dlp/extractor/gab.py

```diff
@@ -22,48 +23,47 @@ class GabTVIE(InfoExtractor):
             'description': None,
             'uploader': 'Wurzelroot',
             'uploader_id': '608fb0a85738fd1974984f7d',
-            'thumbnail': 'https://tv.gab.com/image/61217eacea5665de450d0488',
+            'thumbnail': 'https://tv.gab.com/image/61217eca90b0355f10c99383',
+        }
+    }, {
+        'url': 'https://tv.gab.com/channel/americanflavored/view/downhill-go-kart-race-no-engines-6254b9eaf1b75c149c703b7f',
+        'info_dict': {
+            'id': '6254b9eaf1b75c149c703b7f',
+            'ext': 'mp4',
+            'title': 'Downhill Go Kart Race- No Engines',
+            'description': None,
+            'uploader': 'AmericanFlavored',
+            'uploader_id': '62454a53299da61b33305901',
+            'thumbnail': 'https://tv.gab.com/image/6254b9f31e03ac67731b035a',
         }
     }]
+    _DOMAIN = 'https://tv.gab.com/'
 
     def _real_extract(self, url):
         id = self._match_id(url).split('-')[-1]
         webpage = self._download_webpage(url, id)
         channel_id = self._search_regex(r'data-channel-id=\"(?P<channel_id>[^\"]+)', webpage, 'channel_id')
-        channel_name = self._search_regex(r'data-channel-name=\"(?P<channel_id>[^\"]+)', webpage, 'channel_name')
-        title = self._search_regex(r'data-episode-title=\"(?P<channel_id>[^\"]+)', webpage, 'title')
-        view_key = self._search_regex(r'data-view-key=\"(?P<channel_id>[^\"]+)', webpage, 'view_key')
+        video_info = self._parse_json(self._search_regex(r'(?s)mediaMetadata\:\s+(\{.+?\})', webpage, 'video_info'),
```

Comment:
Unnecessary escaping

Reply:
@pukkandan You mean we dont need transform_source, is it right?

Reply:
No, I don't know if transform_source is needed or not. You'll need to check that. 

I meant this:
```diff
-        video_info = self._parse_json(self._search_regex(r'(?s)mediaMetadata\:\s+(\{.+?\})', webpage, 'video_info'),
+        video_info = self._parse_json(self._search_regex(r'(?s)mediaMetadata:\s+({.+?})', webpage, 'video_info'),